en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-inbound-endpoint-example.md

```diff
@@ -0,0 +1,110 @@
+# ISO8583 Inbound Endpoint Example
+
+In real world financial scenarios happening among thousands of banking systems and networks. In this situation, one system need to act as a message publisher and another system need to capable of receiving them. Once receive the message if further processing actions performed based on the logics which are implemented in the internal system.
+
+The ISO8583 inbound endpoint of WSO2 EI acts as a message consumer. The ISO8583 inbound endpoint supported via the WSO2 EI is a listening inbound endpoint that can consume ISO8583 standard messages.Then converts the messages to XML format, and then injects messages to a sequence in WSO2 EI.
+
+## What you'll build
+
+This scenario demonstrates how the ISO8583 inbound endpoint works as an ISO8583 message consumer. In this scenario, to generate ISO8583 messages we use a sample Java client program.In here inside the banking network functionality simulates using the test client program.
+
+The ISO8583 inbound endpoint listens on port 5000 and acts as a ISO8583 standard message consumer. When a sample Java client connects on port 5000, the ISO8583 inbound endpoint consumes ISO8583 standard messages, converts the messages to XML format, and then injects messages to a sequence into WSO2 EI.
+
+See [ISO8583 connector configuration](ISO8583-connector-configuration.md) for more information.However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 [EI mediators](https://docs.wso2.com/display/EI660/ESB+Mediators). 
```

Comment:
```suggestion
See [ISO8583 connector configuration](ISO8583-connector-configuration.md) for more information. However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 [EI mediators](https://docs.wso2.com/display/EI660/ESB+Mediators). 
```

Shall we update this link to point to 7.0.0 docs?

Reply:
Updated.